src/conf.cpp

```diff
@@ -39,244 +39,138 @@ struct section {
     struct section *next;
 };
 
-ConfFile::~ConfFile()
+ConfFile::ConfFile(const char *filename)
+    : _filename(filename)
+    , _entry(nullptr)
+    , _last_section(nullptr)
+    , _n(0)
+    , _line(0)
```

Comment:
And it would be nice to agree on a implementation and keep in sync the mavlink-router and camera-daemon implementations, unless there's a compelling reason not to do so.